<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color: rgb(255, 255, 255);
}

</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="AdvancedSearch">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Чекбоксы*/
QCheckBox::indicator{
width:24px;
height:24px;
}
QCheckBox::indicator:unchecked{
background-image: url(:/blank/checked-blue.png);
}
QCheckBox::indicator:unchecked:hover, QCheckBox::indicator:unchecked:pressed{
background-image: url(:/blank/blank-red.png);
}
QCheckBox::indicator:checked{
background-image: url(:/checked/blank-blue.png);
}
QCheckBox::indicator:checked:hover, QCheckBox::indicator:checked:pressed{
background-image: url(:/checked/checked-red.png);
}

/* Календарь*/
QCalendarWidget QWidget{
alternate-background-color: #B8E2FF;
}
QCalendarWidget QWidget#qt_calendar_navigationbar{
background-color: rgb(255, 255, 255);
border:2px solid #B8E2FF;
border-bottom: 0px;
}
QCalendarWidget QWidget#qt_calendar_prevmonth, QCalendarWidget QWidget#qt_calendar_nextmonth{
border: none;
qproperty-icon: none;
min-width: 13px;
max-width: 13px;
min-height: 13px;
max-height: 13px;
border-radius: 5px;
background color: transparent;
padding: 5px;
}
QCalendarWidget QWidget#qt_calendar_prevmonth{
margin-top:5px;
margin-left:5px;
margin-bottom:3px;
image: url(:/arrow/arrow2.png);
}
QCalendarWidget QWidget#qt_calendar_nextmonth{
margin-top:5px;
margin-right:5px;
margin-bottom:3px;
image: url(:/arrow/arrow1.png);
}
QCalendarWidget QWidget#qt_calendar_prevmonth:hover, QCalendarWidget QWidget#qt_calendar_nextmonth:hover{
background-color: #B8E2FF;
}
QCalendarWidget QWidget#qt_calendar_yearbutton{
border-radius:5px;
color: #000;
margin-top:5px;
margin-bottom:3px;
border-redis:5px;
font-size:13px;
padding: 0 10px;
}
QCalendarWidget QWidget#qt_calendar_monthbutton{
border-radius:5px;
width:110px;
color: #000;
margin-top:5px;
margin-bottom:3px;
border-redis:5px;
font-size:13px;
padding: 0 2px;
}
QCalendarWidget QWidget#qt_calendar_monthbutton:hover, QCalendarWidget QWidget#qt_calendar_yearbutton:hover{
background-color: #B8E2FF;
}
QCalendarWidget QWidget#qt_calendar_monthbutton:pressed, QCalendarWidget QWidget#qt_calendar_yearbutton:pressed{
background-color: rgb(247, 247, 247);
}
QCalendarWidget QWidget#qt_calendar_yearedit{
margin-top:2px;
width:50px;
color:#000;
font-size:13px;
}
QCalendarWidget QWidget#qt_calendar_yearedit:down-button{
border-image: url(:/arrow/arrow-down.png);
subcontrol-position:left;
}
QCalendarWidget QWidget#qt_calendar_yearedit:up-button{
image: url(:/arrow/arrow-up.png);
subcontrol-position:right;
}
QCalendarWidget QWidget#qt_calendar_yearedit:up-button:hover{
image: url(:/arrow/arrow-up-blue.png);
}
QCalendarWidget QWidget#qt_calendar_yearedit:down-button:hover{
image: url(:/arrow/arrow-down-blue.png);
}
QCalendarWidget QWidget QToolButton::menu-indicator{
image:none;
}
QCalendarWidget QWidget#qt_calendar_calendarview {
outline: 0px;
selection-background-color: rgb(165, 204, 229);
}
</string>
      </property>
      <property name="title">
       <string>Расширенный поиск</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QFrame" name="SatelliteSystem">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="SatelliteLabel">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Спутниковая система</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="SatelliteComboBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <item>
             <property name="text">
              <string>Landsat 5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Landsat 9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sentinel-2</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="SensingPeriod">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="SensingLabelHorizontalLayout">
            <property name="spacing">
             <number>15</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="SensingLabel">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Период зондирования</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="SensingCheckBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="SensingHorizontalLayout">
            <item>
             <widget class="QDateEdit" name="SensingFromDateEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="SensingToDateEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="IngestionPeriod">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="IngestionLabelHorizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="IngestionLabel">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Период приема</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="IngestionCheckBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="IngestionHorizontalLayout">
            <item>
             <widget class="QDateEdit" name="IngestionFromDateEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="IngestionToDateEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>IBeamCursor</cursorShape>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Cloudiness">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="CloudinessLabel">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Облачность: 0% - 100%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Координаты</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
border:none;
background-image: url(:/plus/plus-blue.png);
}
QPushButton:hover:!pressed{
 
	background-image: url(:/plus/plus-red.png);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
background-color: rgb(252, 252, 252);
</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>308</width>
               <height>193</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>172</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Область интереса</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="map" native="true">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="SearchButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Найти</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Результаты</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>290</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollArea{
border:0px;
}
QWidget#scrollAreaWidgetContents{
background-color: rgb(255, 255, 255);
}
</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>288</width>
            <height>638</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/img/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SensingCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SensingFromDateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SensingCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>SensingToDateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IngestionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>IngestionFromDateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IngestionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>IngestionToDateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
